var/spack/repos/builtin/packages/c3d/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class C3d(CMakePackage):
+    """Image processing and conversion tool based on ITK.
+    """
+
+    homepage = "https://github.com/pyushkevich/c3d"
+    git      = "https://github.com/pyushkevich/c3d.git"
+    url      = "https://github.com/pyushkevich/c3d/archive/refs/tags/v1.3.0.tar.gz"
+
+    version('1.3.0', sha256="10277072f24f2bd4bde2f45f00b6a9dd0200400f")
```

Comment:
How did you calculate the sha256?  According to `spack checksum`, it should be:

```suggestion
    version('1.3.0', sha256="bd45482247fa4ac5ab98b3a775b5438390671e3e224a42f73967904b3895050d")
```

Reply:
Weird, perhaps I was initially using a git checkout or something, but not sure how this made it into the commit.  Anyway, fixed.